frontend/array_cmath/include/algebra/array_cmath.hpp

```diff
@@ -110,4 +115,31 @@ using transform3 =
 
 }  // namespace array
 
+namespace plugin {
+
+/// Define the plugin types
+/// @{
+template <concepts::value V>
+struct cmath {
```

Comment:
Given the library and file names, should this not rather be called `array_cmath`?

Given that it's really "just" the storage that is getting defined, if not `array_cmath`, it would have to be `array`. 🤔

Reply:
renamed